weblab/templates/about.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+  <h1>About the Web Lab</h1>
+  <p>
+    The key idea underpinning the Web Lab is that when mathematical and computational models are
+    being developed, a primary goal should be the continuous comparison of those models
+    against experimental data. When computational models are being re-used in new studies, it is similarly
+    important to check that they behave appropriately in the new situation to which you're applying them.
+  </p>
+  <p>
+    To achieve this goal, it's a pre-requisite to be able to replicate <i>in-silico</i> precisely the same protocols used in an experiment of interest.
+    We are therefore developing a language for describing rich 'virtual experiment' protocols,
+    and software for running these on compatible models.
+    This website provides a front-end to a prototype of that language and software.
+  </p>
+  <p>
+    Please <a href="{% url 'experiments:list' %}">browse around</a> and view the results of protocols encoded so far, run on a range of cell models.
+    If you wish to evaluate your own models under these protocols, you will need to <a href="{% url 'accounts:register' %}">register for an account</a> and have it approved by a site admin.
+    By default your own models, protocols and experiments are private so that only you may view them.
+    This can be altered to 'restricted', providing visibility to all logged-in users, or 'public' to publish them to the world.
+  </p>
+  <p>
+    Currently only trusted users may upload new protocols, but if you have one you'd like to add to the system, please do
+    <a href="{% url 'contact' %}">contact us</a>.
+  </p>
+
+  <h2>Further reading</h2>
+  <p>
+    The <a href="https://www.sciencedirect.com/science/article/pii/S0006349515047530">initial paper about the Web Lab</a> was published in <i>Biophysical Journal</i>.
+    <!-- A <a href="https://peerj.com/preprints/1338/">preprint version is available from PeerJ</a>. -->
+    A <a href="https://www.biorxiv.org/content/early/2018/01/31/257683">preprint of our next Web Lab paper</a> sets out our plans for the next two years.
+  </p>
+  <p>
+    For more background on the 'Functional Curation' experiment execution software, our <a href="http://dx.doi.org/10.1016/j.pbiomolbio.2011.06.003">2011 reference publication</a> is a good place to start.
+    We also have a <a href="http://dx.doi.org/10.1016/j.pbiomolbio.2014.10.001">paper on the underlying concept of <i>virtual experiments</i></a>
+    (along with an <a href="https://peerj.com/preprints/273/">open access preprint</a>).
+    Full details about the <a href="https://chaste.cs.ox.ac.uk/trac/wiki/FunctionalCuration">open source software implementing the back-end for this system</a> can be found on the Chaste website.
```

Comment:
Is this still relevant with the new Python back-end?

Reply:
The Python backend is also documented there. But at the moment scrambler is running the original C++ backend.